src/lib.rs

```diff
@@ -217,18 +217,20 @@ pub fn parse(text: &str, options: LiquidOptions) -> Result<Template> {
     parser::parse(&tokens, &options).map(Template::new)
 }
 
-/// Parse a liquid template from a file, returning Result<Template, io::Error>
+/// Parse a liquid template from a file, returning a Result (Template or io Error).
```

Comment:
you don't have to remove the angle brackets, you can simply put it in backticks: 
```
`Result<Template, io::Error>`
```